src/bdfsclient-lib/BdProtocol.h

```diff
@@ -45,7 +45,9 @@ namespace bdfs
     typedef struct
     {
       BdHdr hdr;
-      char data[128];
+      char data1[128];
```

Comment:
Don't hard code the size

Reply:
And also don't use such general fields for anything which is likely to break some time later